.github/workflows/nix.yml

```diff
@@ -11,7 +11,7 @@ on:
      - next*
 
 env:
-  RELOC_VERSION: "2023-04-13T13:31:00Z"
+  RELOC_VERSION: "2024-03-22T09:51:51Z"
```

Comment:
might want to update `.github/workflows/integration-tests.yml` as well. see https://github.com/scylladb/scylla-ccm/blob/e02cfe1e63dc27d86b9c8866120b083bd578844f/.github/workflows/integration-tests.yml#L14